src/logic/operate.js

```diff
@@ -0,0 +1,24 @@
+/* eslint-disable linebreak-style */
+// eslint-disable-next-line import/no-extraneous-dependencies
+import Big from 'big.js';
+
+export default function operate(numberOne, numberTwo, operation) {
+  const one = Big(numberOne);
+  const two = Big(numberTwo);
+  if (operation === '+') {
+    return one.plus(two).toString();
+  }
+  if (operation === '-') {
+    return one.minus(two).toString();
+  }
+  if (operation === 'x') {
+    return one.times(two).toString();
+  }
+  if (operation === '÷') {
+    return one.div(two).toString();
+  }
```

Comment:
- [ ] When I divide a number with `zero` let us say `5/0` normally you have to get an `error` in the screen but currently am getting 
![e](https://user-images.githubusercontent.com/44745855/130041264-8cb594cd-a46a-4582-ae77-5990b97ec9c7.png)

Please fix that

Reply:
ok ok that's true thanks